lxd/task/schedule.go

```diff
@@ -46,6 +48,30 @@ func Every(interval time.Duration, options ...EveryOption) Schedule {
 	}
 }
 
+// Cron returns a Schedule determined by the provided cron expression.
+func Cron(expr string, options ...EveryOption) Schedule {
```

Comment:
We need the SkipFirst behavior. Let's say the cron expression is `* * * * SUN` (runs on Sundays only), and we start the cronjob on a Monday. If we didn't have SkipFirst, it would run on the Monday which it shouldn't.